### ISSUE TITLE
Provide an option to start GameMode whenever a certain app was started (e.g. Steam)

### ISSUE THREAD

**Issue by jgandert** [FeralInteractive/gamemode#67]
**Is your feature request related to a problem? Please describe.**
It's a bit annoying having to edit steam launch options for each and every game. I'd prefer it, if GameMode would start automatically whenever I run Steam or itch.io, since that is when I'll play games. Steam isn't usually open when I don't play.

**Describe the solution you'd like**
An option to specify conditions for starting and ending GameMode.

**Describe alternatives you've considered**
Running my own script that observes whether Steam (or another tool like itch, or an emulator) is started / closed.

**Comment by aejsmith** [comment]
Can you add the LD_PRELOAD to whatever method (e.g. .desktop file) you're using to launch Steam/itch/etc.?

**Comment by CreamyCookie** [comment]
True, I hadn't thought of that. I'll leave it open for now (in case there's still interest for such a feature), but feel free to close if you think it's unnecessary =)
